packages/eslint-plugin/package.json

```diff
@@ -50,6 +50,7 @@
     "debug": "^4.3.4",
     "grapheme-splitter": "^1.0.4",
     "ignore": "^5.2.0",
+    "grapheme-splitter": "^1.0.4",
```

Comment:
looks like a bad merge - added the entry twice!

Reply:
Eek.

Reply:
https://github.com/typescript-eslint/typescript-eslint/commit/99c091e0e3cf38a28aa5e1d1d2b03b602d51aa0a#r98752909